<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDataBase</class>
 <widget class="QWidget" name="TestDataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>273</width>
     <height>235</height>
    </rect>
   </property>
   <property name="title">
    <string>open db</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="wStatusDbOpen" native="true">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 0);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="cbSelectDb">
      <item>
       <property name="text">
        <string>sqlite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sql</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnOpenDb">
      <property name="text">
       <string>open/connect</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QStackedWidget" name="stackSelectDb">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="stackedWidget_2Page1" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Db Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txDbName">
          <property name="text">
           <string>tfri</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>User:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txUser">
          <property name="text">
           <string>root</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="txPassword">
          <property name="text">
           <string>Aa111111</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Ip:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txIp">
             <property name="text">
              <string>157.245.142.72</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbPort">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="keyboardTracking">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>3306</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
